src/main/scala/nl.knaw.dans.easy.deposit/docs/dm/Author.scala

```diff
@@ -46,7 +44,8 @@ case class Author(titles: Option[String] = None,
       case (true, true) => s"$name; ${ organization.getOrElse("") }"
```

Comment:
Regarding the `.filter(...).map(...)` above: could you fit this into a `collect`? Looks like this is a great use case for it and it reduces the number of iterations through the list.